src/help-bot/help-bot.answerer.ts

```
@@ -49,8 +53,7 @@ export interface HelpBotNoReliableSource {
 }
 
 export type HelpBotAnswerResult =
-  | HelpBotAnswerSuccess
-  | HelpBotNoReliableSource;
+  HelpBotAnswerSuccess | HelpBotNoReliableSource;
```

ESLint

[error] Replace `HelpBotAnswerSuccess` with `|·HelpBotAnswerSuccess⏎·`

(prettier/prettier)